egs2/dral/s2st1/steps

```diff
@@ -0,0 +1 @@
+../../../tools/kaldi/egs/wsj/s5/steps
```

Comment:
These might be due to an old version of setup.sh. If you check the current settings, we would always redirect to ../../TEMPLATE/asr1/steps to remove the dependency of kaldi original repo.